cmd/kubelet/app/server.go

```diff
@@ -121,8 +124,52 @@ is checked every 20 seconds (also configurable with a flag).
 HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 (underspec'd currently) to submit a new manifest.`,
 		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
We will need to make sure it's still possible for the Kubelet to re-parse its command line multiple times during its bootstrap, see #56995 and #56171. I'd suppose this can be done with the args []string passed in here but need to make sure.

Reply:
Reparsing is possible. I'll link you to some OpenShift code that does it. In general, be careful about doing it.